src/sentry/api/helpers/group_index/index.py

```diff
@@ -81,6 +81,9 @@ def build_query_params_from_request(
             raise ParseError(detail="Invalid cursor parameter.")
     has_query = request.GET.get("query")
     query = request.GET.get("query", "is:unresolved").strip()
+    if features.has("organizations:issue-priority-ui", organization):
+        query = f"{query} issue.priority:[high, medium]"
```

Comment:
I don't think this is quite right. We only want this to be used when `query` is None, but this looks like it will append the issue priority value to it even if we pass in a custom value